go/nbs/dynamo_manifest.go

```diff
@@ -39,13 +40,14 @@ type ddbsvc interface {
 type dynamoManifest struct {
 	table, db string
 	ddbsvc    ddbsvc
+	lock      addr
```

Comment:
no. it's just a bit smelly. made it hard to debug, but i don't see a concrete problem. mainly because our ordering isn't significant. as long as it isn't causing false lock failures.